Contributing.md

```diff
@@ -0,0 +1,138 @@
+# Contribution Guide
+
+We recommend you to do local setup in a Linux environment. We will soon update the readme for Windows setup.
+
+If you're reading this, you're probably creating a Pull Request or planning to do so and that's great!🥳
+
+# How to Contribute
+
+1. Fork this repository.
+
+2. Clone the forked repository.
+
+   ```bash
+   git clone https://github.com/<your_username>/busify.git
+   ```
+
+3. Navigate to the project directory.
+
+   ```bash
+   cd busify
+   ```
+
+4. Run the Postgres docker container with the command (make sure you have docker installed before running this)
```

Comment:
We can write run the docker container with the command, and also add a link to a resource explaining how to install docker on linux.